podman_hpc/podman_hpc.py

```diff
@@ -96,54 +100,58 @@ def podhpc(ctx, additional_stores, squash_dir, update_conf, log_level):
         ctx.exit()
 
     # set up site configuration object
-    conf = SiteConfig(squash_dir=squash_dir, log_level=log_level)
+    try:
+        conf = SiteConfig(squash_dir=squash_dir, log_level=log_level)
+    except Exception as ex:
+        sys.stderr.write(f"Error: {ex}... Exiting\n")
+        sys.exit(1)
+
     conf.read_site_modules()
     # migrate was here, is that important?
     conf.config_storage(additional_stores)
     conf.config_containers()
     conf.config_env(hpc=True)
 
-    # optionally, save the storage conf
```

Comment:
This is handling the case where a user passes an explicit overwrite flag to save the config file.  Is that no longer being used?

Reply:
If that's the intent, then we can leave the `export_storage_conf()` and `export_containers_conf()` functions in `siteconfig.py` but maybe we should remove the `--update-conf` option flag, since it no longer does anything.

Reply:
looks good to me now